scripts/ci/build_and_test.sh

```diff
@@ -17,6 +17,10 @@ SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
 source ${SCRIPT_DIR}/parse_args.sh "$@"
 
+if [[ -f ~/.cache/.bashrc ]]; then
```

Comment:
why do we need this `.bashrc`?

Reply:
used for HTTPS proxy setting if it exists.

Reply:
I prefer to set proxy using explicit env var.

Reply:
Is there any special reason？ I think we do not want the HTTPS_PROXY to leak?

Reply:
rename `.cache/.bashrc` to `.cache/proxy_config` according to the offline discussion.